docs/build/arm-exception-handling.md

```diff
@@ -167,7 +166,7 @@ When the packed unwind format is insufficient to describe the unwinding of a fun
    |0|20|*X* is a 1-bit field that indicates the presence (1) or absence (0) of exception data.|
    |0|21|*`E`* is a 1-bit field that indicates that information that describes a single epilogue is packed into the header (1) rather than requiring additional scope words later (0).|
    |0|22|*F* is a 1-bit field that indicates that this record describes a function fragment (1) or a full function (0). A fragment implies that there's no prologue and that all prologue processing should be ignored.|
-   |0|23-27|*Epilogue Count* is a 5-bit field that has two meanings, depending on the state of the *`E`* bit:<br /><br /> -   If *`E`* is 0, this field is a count of the total number of epilogue scopes described in section 2. If more than 31 scopes exist in the function, then this field and the *Code Words* field must both be set to 0 to indicate that an extension word is required.<br />-   If *`E`* is 1, this field specifies the index of the first unwind code that describes the only epilogue.|
+   |0|23-27|*Epilogue Count* is a 5-bit field that has two meanings, depending on the state of the *`E`* bit:<br /><br />- If *`E`* is 0, this field is a count of the total number of epilogue scopes described in section 2. If more than 31 scopes exist in the function, then this field and the *Code Words* field must both be set to 0 to indicate that an extension word is required.<br />- If *`E`* is 1, this field specifies the index of the first unwind code that describes the only epilogue.|
```

Comment:
This follows the "convention" in this topic where there is no space between the `br` element and the hyphen, with one after.